src/com/mendix/widget/GoogleMaps/components/__tests__/Map.spec.ts

```diff
@@ -0,0 +1,201 @@
+import { shallow } from "enzyme";
+import GoogleMap from "google-map-react";
+import { DOM, createElement } from "react";
+
+import { Map, MapProps } from "../Map";
+import { Marker } from "../Marker";
+
+import {
+    EventMock, GeocoderLocationType, GeocoderMock, GeocoderStatus, LatLngBoundsMock,
+    LatLngMock, MapsMock, MarkerMock, MockGoogle
+} from "../../../../../../../tests/mocks/GoogleMaps";
```

Comment:
The typescript compiler likes it, however the webpack is complaining.
I solved it in the webpack.config.js with:
```
resolve: {
    alias: {
        "tests": path.resolve(__dirname, "./tests")
    },
}
```
Or is there a better way of doing it?